modules/lang-painless/build.gradle

```diff
@@ -169,6 +169,12 @@ dependencies {
 String grammarPath = 'src/main/antlr'
 String outputPath = 'src/main/java/org/elasticsearch/painless/antlr'
 
+spotless {
+  java {
+    targetExclude "${outputPath}/*.java"
```

Comment:
Ug, we really should be putting generated source elsewhere instead of mixing it with the main sourceset. Let's add a `TODO` here.